common/txmgr/txmgr.go

```diff
@@ -472,12 +472,12 @@ func (b *Txm[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) CreateTran
 		}
 	}
 
-	err = b.txStore.CheckTxQueueCapacity(txRequest.FromAddress, b.txConfig.MaxQueued(), b.chainID, qs...)
+	err = b.txStore.CheckTxQueueCapacity(txRequest.FromAddress, b.txConfig.MaxQueued(), b.chainID)
 	if err != nil {
 		return tx, errors.Wrap(err, "Txm#CreateTransaction")
 	}
 
-	tx, err = b.txStore.CreateTransaction(txRequest, b.chainID, qs...)
+	tx, err = b.txStore.CreateTransaction(txRequest, b.chainID)
```

Comment:
Since this call is ultimately coming from an external caller, we should take a param ctx from external caller, and pass the ctx all the way to the evmTxStore.CreateTx(), and in to the DB call.
Thus, if the caller wants to abandon their call, they should be able to.